src/frontend/contexts/userContext.js

```diff
@@ -12,6 +12,12 @@ const UserProvider = ({children}) => {
                 return { ...state, ...action.payload };
             case 'LOGOUT':
                 return { ...state , ...user };
+            case 'ADDTOHISTORY': 
```

Comment:
Just a suggestion to use this naming convention for better readability - ADD_TO_LIKED